src/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
 
   <PropertyGroup>
-    <VersionPrefix>4.0.0-preview</VersionPrefix>
+    <VersionPrefix>4.0.2-preview</VersionPrefix>
```

Comment:
🤔 I see 4.0.2 in master already.

Reply:
Correct; I just set it to 4.0.2 to get rid of the merge conflict.  But wouldn't this go in 4.1.0 anyway?  Probably won't merge it quite yet in case there's any other changes we want in 4.0.2, such as #2416

Reply:
If this can go in 4.0.2 then it can be merged as-is.

Reply:
My question is why I see this diff at all.

Reply:
Because it isn't merged with master.  GitHub is allowing it because the files are now identical, so it can be merged automatically without creating a conflict.

Reply:
Sorry, but I don't understand. I see `4.0.2-preview` in master. Your PR targets master as well. So why I see this:
![изображение](https://user-images.githubusercontent.com/21261007/111828872-fc3e2780-88fc-11eb-9817-36a3ea9634b0.png)

Reply:
Because this branch was "branched" off of the master branch when it said 4.0.0-preview.  GitHub is showing changes that I've made to this branch as it compares to when it was "branched", not as compared to what it is now.  Does that make sense?

Reply:
🤦‍♂️ Yes, thanks!